styles.css

```diff
@@ -0,0 +1,1056 @@
+      display: grid;
+      grid-template-columns: 1fr 1fr 1fr 1fr;
+      grid-template-rows: 1fr 1fr;
+      gap: 10px;
+      margin-top: 5px;
+      margin-bottom: 2em;
+      padding-top: 30px;
+      word-spacing: normal;
+      font-family: Georgia, 'Times New Roman', Times, serif;
+      font-weight: bold;
+  }
+  .deals-links-boxes a {
+      text-decoration: none;
+      color: silver;
+  }
+  
+  .latest-deals-box1-img img {
+      width: 270px;
+      height: 150px;
+  }
+  
+  
+  
+  
+   /* footer */
+  
+  .footer {
+      width: 100%;
+      text-align: center;
+      background-color: #131313;
+      height: 300px;
+  }
+  
+  .footer a {
+      padding: 5px 20px;
+      text-decoration: none;
+      color: #FFFFFF;
+  }
+  
+  .footer-links {
+      height: 200px;
+      width: 100%;
+  }
+  
+  .footer-social-links {
+      display: flex;
+      justify-content: space-around;
+      position: block;
+      width: 500px;
+      align-items: center;
+      text-align: center;
+      color: #FFFFFF;
+      padding-top: 50px;
+      margin-left: 330px;
+  }
+  
+  .footer-social-links a {
+      width: 100%;
+      background: #333;
+      border-radius: 50%;
+      display: block;
+      height: 40px;
+      padding: 12px;
+      text-align: center;
+      width: 40px;
+  }
+  
+  
+  .footer-menu-links {
+      display: flex;
+      justify-content: space-around;
+      position: block;
+      width: 300px;
+      align-items: center;
+      text-align: center;
+      margin-left: 435px;
+      padding: 50px;
+  }
+  
+  .footer-menu-links a {
+      text-decoration: none;
+      font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+      color: darkgray;
+      font-weight: lighter;
+      font-size: 17px;
+      word-spacing: 2px;
+  }
+  
+  .footer-legal {
+      background-color: #000000;
+      color: darkgray;
+      font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+      font-size: 12px;
+      padding-top: 30px;
+      height: 100px;
+      width: 100%;
+  }
+  
+  .footer-legal div {
+      margin-right: 60px;
+  }
+  
+  .stackpath img {
+      width: 60px;
+      height: 30px;
+      padding-top: 20px;
+  }
+  
+  .tnw img {
+      width: 40px;
+      height: 30px;
+      padding-top: 20px;
+  }
+  
+  
+  
+  
+  
+  
+  
+  
+   /* responsive web design */
+  
+  /* Extra small devices (phones, 763px and down) */
+   @media only screen and (max-width: 768px) {
+      .main {
+        display: flex;
+        flex-direction: column;
+        width: 100%;
+      } 
+
+      .footer-social-links {
+        max-width: 100%;
+      }
+
+      .latest-news {
+        width: 100%;
+      }
+
+      .latest-news-details {
+        width: 100%;
+      }
+
+      .box-wrapper {
+        width: 100%;
+        padding: 0 0;
+      }
+      .latest-box {
+        width: 100%;
+      }
```

Comment:
selectors on lines 889 and 897 have the same style definition. Group selectors with the same style definition.